Packs/SimpleDebugger/Scripts/SimpleDebugger/SimpleDebugger.py

```diff
@@ -0,0 +1,373 @@
+        return c
+
+    def SdbgSetCurrentLineno(self, frame: FrameType, c: str) -> bool:
+        newcode = False
+        if frame.f_lineno not in self.code:
+            self.code[frame.f_lineno] = c
+            self.code = OrderedDict(sorted(self.code.items()))
+            newcode = True
+        else:
+            self.code[frame.f_lineno] = c
+        if self.prevlineno != 0:
+            self.code[self.prevlineno] = self.code[self.prevlineno].lstrip("**").rstrip("**")
+        self.prevlineno = frame.f_lineno
+        return newcode
+
+    def SdbgCommand(self, frame: FrameType, breakpnt: bool) -> bool:
+        if breakpoint is False:
+            return False
+        fields = execute_command("getIncidents", {"id": self.incid})['data'][0]['CustomFields']
+        cmd = fields['simpledebuggercmd'].split(" ", 1)
+        command = cmd[0].strip()
+        if command == "continue":
+            self.SdbgLog("Continue execution")
+            self.stepmode = False
+            breakpnt = False
+        elif command == "step":
+            self.stepmode = True
+            breakpnt = False
+        elif command == "print":
+            self.SdbgLog(f"Print {cmd[1]}")
+            self.SdbgPrintFunc(cmd[1])
+        elif command == "stop":
+            self.SdbgLog("Stop debugger and exit")
+            self.SdbgTraceOff()
+            sys.exit()
+        elif command == "break":
+            self.SdbgLog("Setting breakpoints")
+            self.SdbgSetBreakpoint(cmd[1].split(","))
+        elif command == "nolog":
+            self.logmode = False
+        elif command == "log":
+            self.logmode = True
+
+        execute_command("setIncident", {'simpledebuggercmd': ""})
+        return breakpnt
+
+    def SdbgException(self):
+        self.SdbgLog(f"Exception thrown at line: {self.code[self.prevlineno].replace(' ', '')}, printing local variables")
+        self.SdbgPrintLocals()
+        self.SdbgTraceOff()
+
+    def SdbgBreak(self, frame: FrameType, event: str) -> bool:
+        if frame.f_lineno in self.linebreak:
+            self.SdbgLog(f"Breakpoint at line > {frame.f_lineno}")
+            return True
+        if event == "call" and frame.f_code.co_name in self.funcbreak:
+            self.SdbgLog(f"Breakpoint at function > {frame.f_code.co_name}")
+            return True
+        if self.stepmode:
+            return True
+        return False
+
+    def SdbgExclude(self, frame: FrameType, event: str) -> bool:
+        if frame.f_code.co_name in self.silent and event == "line":
+            return True
+        if frame.f_code.co_name in self.exclude:
+            return True
+        if frame.f_code.co_name.startswith("_"):
+            return True
+        if frame.f_code.co_name.startswith("<"):
+            return True
+        if frame.f_code.co_name.startswith("Sdbg"):
+            return True
+        return False
+
+    def SdbgTrace(self, frame: FrameType, event: str, arg: Any):
```

Comment:
Please rename, it since our linters fail on it. 

```suggestion
    def SdbgTrace(self, frame: FrameType, event: str, _arg: Any):
```